package-lock.json

```diff
@@ -2051,9 +2051,9 @@
       }
     },
     "node_modules/bento-charts": {
-      "version": "2.8.0",
-      "resolved": "https://registry.npmjs.org/bento-charts/-/bento-charts-2.8.0.tgz",
-      "integrity": "sha512-Jzjyhh+iXme1NdEZHRHjJIgqaL54+X99AWscrO1Yxkb9yf7YlSV2UaDf9ZxNd0PIt7AeOl24Us0HuhDojIP30A==",
+      "version": "2.9.0",
```

Comment:
update the minimum version in package.json to be 2.9.0 too, since we need the new features